src/common/components/NFTCard/NFTCard.tsx

```diff
@@ -159,20 +163,23 @@ const NFTCard = ({
           <div
             className={`absolute h-1.5 inline-block rounded-r-lg ${styles.bgPercentage}`}
             style={{ width: `${percentage}%` }}
-          ></div>
+          />
         </div>
       ) : null}
       <Link to={detailUrl} className='cursor-pointer w-full'>
         <div
           className={cn(
-            'relative',
+            'relative overflow-hidden',
             isPortfolio && 'h-220px',
             !isPortfolio && imageHeightClass,
           )}
```

Comment:
I think it's better to use isPortfolio? 'h-220px': imageHeightClass. Can we change this?

Reply:
Fixed, though I don't think it is better and this wasn't changed by this PR

Reply:
okay I can see this is not belongs to PR. Anyway thank you!